install_files/ansible-base/roles/app/tasks/install_and_harden_apache.yml

```diff
@@ -2,7 +2,7 @@
 - name: Install apache packages.
   apt:
     pkg: "{{ apache_packages }}"
-    state: latest
+    state: present
```

Comment:
Changed this to state=present to avoid yet another `apt-get update`. Earlier in the playbook run, _all_ packages are updated to via apt safe-upgrade (dist-upgrade, as of #5793), so there's no need to update the lists again.